source/code/plugins/patch_management_lib.rb

```diff
@@ -230,8 +254,8 @@ def self.transform_and_wrap(inventoryXMLstr, host, time, force_send_run_interval
         availableUpdatesXML = instancesXML.select { |instanceXML| isAvailableUpdateInstanceXML(instanceXML) }
 
         # Convert to xml to hash/json representation
-        installedPackages = installedPackagesXML.map { |installedPackage|  installedPackageXMLtoHash(installedPackage, osShortName)}
-        availableUpdates = availableUpdatesXML.map { |availableUpdate|  availableUpdatesXMLtoHash(availableUpdate, osShortName)}
+        installedPackages = installedPackagesXML.map! { |installedPackage|  installedPackageXMLtoHash(installedPackage, osShortName)}
```

Comment:
Offline discussed with Robbie. This is a bug in existing code. Not related to this code change. A fix will be issued in a separate code change. Bug 53243 - Need move XML to JSON transformation from filter to source to resolve memory issue was opened to track the fix.